### DISCUSSION TITLE
Rain

### DISCUSSION THREAD

**Discussion by doqc** [primefaces/community#528] [PrimeFaces Templates]
Hello, I'm new to using primefaces, I bought the rain template, I'm using eclipse, I would like to know how I do, a step by step, to get the template to work.

**Comment by NicolaIsotta** [comment]
https://primefaces.org/rain/documentation.xhtml

**Comment by mertsincan** [comment]
Also, these videos can help you to integrate layouts into your project;
https://www.dropbox.com/s/gat9a4ym373miyv/import_Barcelona_Theme_into_a_project.mp4?dl=0
https://www.dropbox.com/s/2p74g63wldbover/import_Ultima_Theme_into_a_project.mp4?dl=0

If you have any problems, please feel free to report us.
Best Regards,
